cdap-app-templates/cdap-etl/cdap-etl-lib/pom.xml

```diff
@@ -190,7 +185,8 @@
         <extensions>true</extensions>
         <configuration>
           <instructions>
-            <Export-Package>co.cask.cdap.templates.etl.realtime.*;co.cask.cdap.templates.etl.batch.*;co.cask.cdap.templates.etl.transforms.*</Export-Package>
+            <Export-Package>co.cask.cdap.templates.etl.realtime.*,co.cask.cdap.templates.etl.batch.*,
+              co.cask.cdap.templates.etl.transforms.*,co.cask.cdap.templates.etl.common</Export-Package>
```

Comment:
They have different effect. With `.*` all sub-packages are included. Without, then only the one mentioned is included.